packages/rest-typings/src/v1/federation/FederationVerifyMatrixIdProps.ts

```diff
@@ -13,6 +13,9 @@ const FederationVerifyMatrixIdPropsSchema = {
 			type: 'array',
 			items: { type: 'string' },
 			uniqueItems: true,
+			additionalItems: false,
```

Comment:
I think there's still an error happening here: ` Error: strict mode: "additionalItems" is ignored when "items" is not an array of schemas` 😬